charts/incubator/oscam/values.yaml

```
@@ -0,0 +1,28 @@
+# Default values for oscam
+
+image:
+  repository: ghcr.io/linuxserver/oscam
+  pullPolicy: IfNotPresent
+  tag: latest
+
+env:
+  TZ: UTC
+  PUID: "1000"
+  # currently needed for working with /dev/ttyXYZ devices
+  # see: https://github.com/linuxserver/docker-oscam/issues/21
+  PGID: "20"
```

Indeed, lets keep group on 20 for now!
```suggestion
env:
  TZ: UTC
  PUID: "568"
  # currently needed for working with /dev/ttyXYZ devices
  # see: https://github.com/linuxserver/docker-oscam/issues/21
  PGID: "20"
```